api/controllers/UserController.js

```diff
@@ -100,8 +96,27 @@ module.exports = {
   },
 
   logout: function (req, res) {
-    delete req.session.usr;
-    req.session.loggedIn = false;
-    return res.ok();
+    req.session.destroy(function afterDestroy() {
+      return res.ok();
+    });
```

Comment:
This seems to be the "official" way to destroy a session, cobbled together by various stackoverflow searches. Also seemed safer than what you were doing previously.